README.md

```diff
@@ -1,2 +1,282 @@
-# Hello-world
-A test repository to practice and master tools and best practices such as Linter, Github flow.Done for Microverse program second day exercise.
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <!-- <img src="murple_logo.png" alt="logo" width="140"  height="auto" /> -->
+  <!-- <br/> -->
+
+  <h3><b>Linters and Github flow practice</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello-world] <a name="practice tools like gitub flow and linters and best practices"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[hello-world]** is a...
```

Comment:
- [ ] Kindly customize your README with the actual information of your project. Remember that having professional code documentation will make your projects more attractive to people that want to collaborate on them and also will provide better guidance on how to interact with them. Please consider modifying all the relevant sections of your README (e.g. Description, Getting started, etc)! 🤓 🛠️

Reply:
pull request for practicing GitHub best practices specifically the pull request process and commenting best practice, added a more specific comment, and added linters to the local environment.